integrated_channels/cornerstone/views.py

```diff
@@ -106,6 +106,7 @@ def get(self, request, *args, **kwargs):
         IntegratedChannelAPIRequestLogs = apps.get_model(
             "integrated_channel", "IntegratedChannelAPIRequestLogs"
         )
+        enterprise_customer_uuid = request.GET.get('ciid')
```

Comment:
is this intentional? 🤔

Reply:
i checked out on master 
and it was missing there.